aten/src/ATen/native/native_functions.yaml

```diff
@@ -618,7 +618,7 @@
 
 - func: embedding_backward(Tensor grad, IndexTensor indices, int64_t num_weights, int64_t padding_idx, bool scale_grad_by_freq, bool sparse) -> Tensor
 
-- func: embedding_dense_backward(Tensor grad, IndexTensor indices, int64_t num_weights, int64_t padding_idx, bool scale_grad_by_freq) -> Tensor
+- func: embedding_dense_backward(Tensor grad_f, IndexTensor indices, int64_t num_weights, int64_t padding_idx, bool scale_grad_by_freq) -> Tensor
```

Comment:
Does grad not work? If not, from looking at the other declarations in native_functions.yaml, I think the more common name for this might be grad_output.

Reply:
From this [line](https://github.com/pytorch/pytorch/blob/a6f582c48363a6d7f27cbccf50a206070029a6c4/tools/autograd/derivatives.yaml#L74-L78), I guess the name grad won't work. But yes, looking at other functions now, `grad_output` looks to be the common one. Thank You.